src/utils/redis.js

```diff
@@ -0,0 +1,63 @@
+const { promisify } = require('util');
+const redis = require('redis');
+const jwt = require('jsonwebtoken');
+const { GeneralError } = require('@feathersjs/errors');
+
+let redisClient = false;
+
+function initializeRedisClient(app) {
+	const redisUrl = app.Config.data.REDIS_URI;
+	if (redisUrl) {
+		try {
+			redisClient = redis.createClient({
+				url: redisUrl,
+			});
+		} catch (err) {
+			throw new GeneralError('redis connection failed!', err);
+		}
+	}
+}
+
+function getRedisClient() {
+	return redisClient;
+}
+
+const redisGetAsync = (...args) => {
+	if (redisClient) return promisify(redisClient.get).apply(redisClient, args);
```

Comment:
Könnte man hier ein sicher-zugriff-und-map-zu-promise helper machen? 

(...args) => {
	if (redisClient) return promisify(redisClient.<generischer part>).apply(redisClient, args);
	throw new GeneralError('no redis connection. check for this via getRedisClient()');
};

Reply:
Dann könnte man ein test für den machen und hätte das für alle abgedeckt.